### PR TITLE
Changing the endpoint of TestSuite

### PR DESCRIPTION
**Pull request by prawalagarwal**
Description: The current api endpoint for TestSuite does not return the data in tree structure required for the MultiSelectTree Control
Solution: Changing the endpoint
Testing: Manual

### CODE REVIEW

File: Tasks/RunDistributedTests/task.json [hunk 1]
```diff
@@ -245,9 +245,8 @@
      },
      {
          "target": "testSuite",
-         "endpoint":"/$(system.teamProject)/_apis/test/plans/$(testPlan)/suites",
-         "selector":"jsonpath:$.value[*].name",
-         "keySelector":"jsonpath:$.value[*].id",
```
Comment (omeshp) [inline Tasks/RunDistributedTests/task.json]: Can the keySelector functionality be removed from the controller now?

Comment (prawalagarwal) [inline Tasks/RunDistributedTests/task.json]: This is removed only for MultiSelectTree control. This functionality is needed for two other controls and hence cannot be removed!

**Comment by kaadhina** [comment]
Looks fine

File: Tasks/RunDistributedTests/task.json [hunk 2]
```diff
@@ -245,9 +245,8 @@
...
+         "endpoint":"/$(system.teamProject)/_apis/test/plans/$(testPlan)/suites?$asTreeView=true",
+         "selector":"jsonpath:$.value[*]",
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/task.json]: are we not combining the inputs we added in preview to main control.

Comment (acesiddhu) [inline Tasks/RunDistributedTests/task.json]: as per atin we will do that merge next after customer validation. ignore the comment

**Comment by acesiddhu** [comment]
this looks fine

**Pull request merged**